exporter/coralogixexporter/factory.go

```diff
@@ -32,6 +32,7 @@ func NewFactory() component.ExporterFactory {
 		createDefaultConfig,
 		component.WithTracesExporterAndStabilityLevel(createTraceExporter, stability),
 		component.WithMetricsExporterAndStabilityLevel(createMetricsExporter, stability),
+		component.WithLogsExporterAndStabilityLevel(createLogsExporter, stability),
```

Comment:
Should the stability level be different for logs as it is the initial implementation?

Reply:
Changed to alpha, thanks!